client/verta/verta/_uac/_organization.py

```diff
@@ -0,0 +1,22 @@
+from ._role import Role
+from verta._protos.public.uac import GroupV2_pb2 as _Group
+from verta._protos.public.uac import RoleV2_pb2 as _Role
```

Comment:
I would actually recommend using `..._pb2` so it's clear it is a protobuf module (older parts of the client use `import ... as ...`, which I know is confusing and I intend to update in the future)

```suggestion
from verta._protos.public.uac import GroupV2_pb2
from verta._protos.public.uac import RoleV2_pb2
```